suzieq/poller/worker/nodes/node.py

```diff
@@ -690,7 +718,7 @@ async def _local_gather(self, service_callback: Callable,
                 self.current_exception = e
                 result.append(self._create_error(cmd))
 
-        await service_callback(result, cb_token)
+        await self.exec_svc_callback(service_callback, result, cb_token)
```

Comment:
Should the local gather release the semaphore? It never acquires it

Reply:
Not for now